go/contacts/phonebook.go

```diff
@@ -96,6 +96,25 @@ func ResolveAndSaveContacts(mctx libkb.MetaContext, provider ContactsProvider, c
 		mctx.Warning("error retrieving synced contacts; continuing: %s", err)
 	}
 
+	if len(newlyResolved) > 0 {
+		resolutionsForPeoplePage := make([]ContactResolution, len(newlyResolved))
+		for i, contact := range newlyResolved {
+			resolutionsForPeoplePage[i] = ContactResolution{
+				Description: fmt.Sprintf("%s (%s)", contact.ContactName,
+					contact.Component.ValueString()),
+				ResolvedUser: keybase1.User{
+					Uid:      contact.Uid,
+					Username: contact.Username,
+				},
+			}
+		}
+		err = SendContactResolutionToServer(mctx, resolutionsForPeoplePage)
+		if err != nil {
+			mctx.Warning("Could not add resolved contacts to people page: %v"+
+				"; returning contacts anyway", err)
```

Comment:
its fine if its more than 80 chars

Reply:
but if you want to do this i'd use backticks for multiline strings (works great except when you have to insert backticks into the string)